tests/validation/tests/v3_api/test_create_ha.py

```diff
@@ -273,6 +282,24 @@ def set_route53_with_aks_ingress():
     time.sleep(60)
 
 
+def set_route53_with_gke_ingress():
```

Comment:
the method `set_route53_with_eks_ingress`, `set_route53_with_aks_ingress` and `set_route53_with_gks_ingress` are pretty much the same. Can we combine them as one method?

Reply:
Good point I refactored this to one method.